### PR TITLE
Bump Polyamorous to 1.3.2 for Rails 5.2 support

### PR DESCRIPTION
**Pull request by seanfcarroll**
Please add comments / review before this is merged.

### CODE REVIEW

File: ransack.gemspec [hunk 1]
```diff
@@ -6,8 +6,8 @@ Gem::Specification.new do |s|
   s.name        = "ransack"
   s.version     = Ransack::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Ernie Miller", "Ryan Bigg", "Jon Atack"]
-  s.email       = ["ernie@erniemiller.org", "radarlistener@gmail.com", "jonnyatack@gmail.com"]
```
Comment (seanlinsley) [inline ransack.gemspec]: My understanding of `authors` is that it represents the people who wrote a substantial portion of the library. I could certainly be wrong, though.

On the other extreme, ActiveAdmin only lists Greg Bell as an author:

screen shot 2017-12-31 at 2 58 10 pm

Comment (seanfcarroll) [reply ransack.gemspec]: Any chance the code could be tested @seanlinsley ?

Comment (bbonislawski) [reply ransack.gemspec]: Agreed. Do not change authors. I wonder even if it's required to change anything since ~1.3 means 1.3.2 too

Comment (seanlinsley) [reply ransack.gemspec]: Unless there's a reason to prevent applications from using 1.3.0 and 1.3.1, @bbonislawski appears to be right.

Comment (ernie) [reply ransack.gemspec]: Given there's no "maintainers" section, I'm fine with adding an author, personally.

Comment (seanfcarroll) [reply ransack.gemspec]: Thanks @ernie

**Comment by seanfcarroll** [comment]
@bbonislawski it was added as 1.3.2 as it wasn't picking up the changes in Polyamorous. Perhaps merge this change with https://github.com/activerecord-hackery/ransack/pull/859 and we can try for a 5.2 - friendly release.

I think we should also consider putting Ransack on a bit of a diet, and dropping older Rails support. https://github.com/activerecord-hackery/ransack/issues/857 what do you think?

**Comment by seanlinsley** [comment]
👍 (I posted my thoughts on #857)

**Comment by seanfcarroll** [comment]
OK to merge this?

**Comment by seanlinsley** [comment]
Looks good to me

**Pull request merged**